tsconfig.json

```diff
@@ -2,9 +2,13 @@
   "extends": "@vtex/tsconfig",
   "compilerOptions": {
     "lib": ["dom"],
-    "target": "ES5",
+    "target": "ES2015",
```

Comment:
Curiosity: how do _you_ decide which version to use?

Reply:
this setting is for development only, so it doesn't matter which target we use here, the `tsconfig.bundle.json` uses `ES3` which is the minimum required in Checkout, and since we will start using this module there it makes sense to lower the target to `ES3`

Reply:
Fair enough

> so it doesn't matter which target we use here

then why change? it's required for something that you'll do in the other branch?